src/docfx/Models/BuildCommandOptions.cs

```diff
@@ -60,6 +60,9 @@ internal class BuildCommandOptions : ICanPrintHelpMessage, ILoggable
         [Option('f', "force", HelpText = "Force re-build all the documentation")]
         public bool ForceRebuild { get; set; }
 
+        [Option("forcePostProcess", HelpText = "Force to re-process the documentation in post processors. It will be cascaded from force option.")]
+        public bool ForcePostProcess { get; set; }
```

Comment:
I see, it's okay

---
In reply to: [94204870](https://github.com/dotnet/docfx/pull/1091#discussion_r94204870) [](ancestors = 94204870,94204150)